### PR TITLE
feat: target page changed message is different when cards UI is enabled

### PR DESCRIPTION
**Pull request by waabid**
#### Description of changes

The changes to the header bar require we change the message in the target page changed dialog (based on whether the feature flag is enabled).

image

#### Pull request checklist

- [n/a] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [x] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/components/adhoc-static-test-view.tsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ export interface AdhocStaticTestViewProps {
     configuration: VisualizationConfiguration;
     content?: ContentReference;
     guidance?: ContentReference;
+    featureFlagStoreData: FeatureFlagStoreData;
```
Comment (RobGallo) [inline src/DetailsView/components/adhoc-static-test-view.tsx]: Is this allowed to be nullable like the properties above it?

File: src/DetailsView/components/target-page-changed-view.tsx [hunk 2]
```diff
@@ -11,11 +13,16 @@ export interface TargetPageChangedViewProps {
     visualizationType: VisualizationType;
     displayableData: DisplayableVisualizationTypeData;
     toggleClickHandler: (event) => void;
+    featureFlagStoreData: FeatureFlagStoreData;
 }
 
 export const TargetPageChangedView = NamedFC<TargetPageChangedViewProps>('TargetPageChangedView', props => {
     const { title = '', toggleLabel = '', subtitle } = props.displayableData;
 
+    const oldText = 'The target page was changed. Use the toggle to enable the visualization in the current target page.';
```
Comment (RobGallo) [inline src/DetailsView/components/target-page-changed-view.tsx]: Could we name this something more informative than 'oldText'? :)

File: src/DetailsView/components/target-page-changed-view.tsx [hunk 3]
```diff
@@ -11,11 +13,16 @@ export interface TargetPageChangedViewProps {
...
+    const cardsUIText = 'The target page has changed. Use the start over button to scan this new target page.';
```
Comment (RobGallo) [inline src/DetailsView/components/target-page-changed-view.tsx]:
```suggestion
    const cardsUIText = 'The target page has changed. Use the start over button to scan the new target page.';
```

File: src/DetailsView/components/target-page-changed-view.tsx (hunk 2 above)
Comment (waabid) [reply src/DetailsView/components/target-page-changed-view.tsx]: Good point! I wasn't sure what to rename it to: any ideas?

Maybe toggleBasedText and startOverBasedText?

**Comment by dbjorge** [comment]
This doesn't appear to be working when I pull and build locally with merge from master, marking do not merge while I investigate why

**Comment by dbjorge** [comment]
False alarm, looks good to me (I was seeing bad results because I hadn't updated my dev extension install to account for #1614)

**Pull request merged**